task3/errorCodes.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
+
+enum errorCodes

```

Comment:
Типы именуются с заглавной всегда, 
```suggestion
enum ErrorCodes
```